tests/integration/tools/impl/test_file_system_edit.py

```
@@ -0,0 +1,397 @@
+        edits=[
+            EditOperation(
+                old_content="        if True:\n            print('hello')\n            return 42",
+                new_content="        if True:\n            print('world')\n            return 99",
+            )
+        ],
+    )
+
+    await run_tool(create_test_graph([edit_file]), state, agent_context)
+
+    content = (temp_dir / "whitespace.py").read_text()
+    assert "print('world')" in content and "return 99" in content
```

Ruff (0.14.5)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)